CONTRIBUTING.md

```diff
@@ -33,8 +33,10 @@ To contribute:
     [clang-tidy](https://clang.llvm.org/extra/clang-tidy/) tools can be
     integrated into your workflow to ensure formatting and stylistic
     compliance of your changes. To avoid code formatting misalignment with
-    GitHub Actions check we recommend using 10 version of clang-format tool
-    (default version on Ubuntu 22.04).
+    the GitHub Actions check we recommend using the same version of clang-format
+    as used in the CI. See `.github/workflows/pr-code-format.yml` for the
+    current version, which at the time of writing this document was
+    clang-format-18.
```

Comment:
Don't think that the last part with "... at the time of writing this document ..." is necessary.
```suggestion
    as used in the CI. See `.github/workflows/pr-code-format.yml` for that.
```

Reply:
Ending the sentence with "for that" is awkward. I'll use "See `.github/workflows/pr-code-format.yml` for the current version".